preview/upgrade/autoosupgrade-doc.md

```diff
@@ -48,6 +48,10 @@ The following SKUs are currently supported (more will be added):
 | MicrosoftWindowsServer  | WindowsServer | 2012-R2-Datacenter | latest   |
 |  MicrosoftWindowsServer | WindowsServer | 2016-Datacenter    | latest   |
 | Canonical               | UbuntuServer  | 16.04-LTS          | latest  |
+|MicrosoftCblMariner      | Cbl-Mariner   | cbl-mariner-1      | latest   |
+|MicrosoftCblMariner      | Cbl-Mariner   | 1-gen2             | latest   |
+|MicrosoftCblMariner      | Cbl-Mariner   | cbl-mariner-2      | latest   |
+|MicrosoftCblMariner      | Cbl-Mariner   | cbl-mariner-2-gen2 | latest   |
```

Comment:
It does not have the full list. Suggest to add the complete list:

canonical:0001-com-ubuntu-server-focal:20_04-lts,_
        canonical:0001-com-ubuntu-server-focal:20_04-lts-gen2,_
        canonical:ubuntuserver:18.04-lts,_
        canonical:ubuntuserver:18_04-lts-gen2,_
        microsoftcblmariner:cbl-mariner:cbl-mariner-1,_
        microsoftcblmariner:cbl-mariner:1-gen2,_
        microsoftcblmariner:cbl-mariner:cbl-mariner-2,_
        microsoftcblmariner:cbl-mariner:cbl-mariner-2-gen2,_
        microsoftwindowsserver:windowsserver:2012-r2-datacenter,_
        microsoftwindowsserver:windowsserver:2016-datacenter,_
        microsoftwindowsserver:windowsserver:2016-datacenter-gensecond,_
        microsoftwindowsserver:windowsserver:2016-datacenter-gs,_
        microsoftwindowsserver:windowsserver:2016-datacenter-smalldisk,_
        microsoftwindowsserver:windowsserver:2016-datacenter-with-containers,_
        microsoftwindowsserver:windowsserver:2016-datacenter-with-containers-gs,_
        microsoftwindowsserver:windowsserver:2019-datacenter,_
        microsoftwindowsserver:windowsserver:2019-datacenter-core,_
        microsoftwindowsserver:windowsserver:2019-datacenter-core-with-containers,_
        microsoftwindowsserver:windowsserver:2019-datacenter-gensecond,_
        microsoftwindowsserver:windowsserver:2019-datacenter-gs,_
        microsoftwindowsserver:windowsserver:2019-datacenter-smalldisk,_
        microsoftwindowsserver:windowsserver:2019-datacenter-with-containers,_
        microsoftwindowsserver:windowsserver:2019-datacenter-with-containers-gs,_
        microsoftwindowsserver:windowsserver:2022-datacenter,_
        microsoftwindowsserver:windowsserver:2022-datacenter-smalldisk,_
        microsoftwindowsserver:windowsserver:2022-datacenter-azure-edition,_
        microsoftwindowsserver:windowsserver:2022-datacenter-core,_
        microsoftwindowsserver:windowsserver:2022-datacenter-core-smalldisk,_
        microsoftwindowsserver:windowsserver:2022-datacenter-g2,_
        microsoftwindowsserver:windowsserver:2022-datacenter-smalldisk-g2